docs/UsersGuide/source/ConfigParameters.inc

```diff
@@ -233,106 +668,347 @@ Set parameters associated with the fixed (i.e. static) files.  For the main NOAA
 
-Customized Post Configuration Parameters
-========================================
-``USE_CUSTOM_POST_CONFIG_FILE``: (Default: “FALSE”)
-   Flag that determines whether a user-provided custom configuration file should be used for post-processing the model data. If this is set to "TRUE", then the workflow will use the custom post-processing (UPP) configuration file specified in ``CUSTOM_POST_CONFIG_FP``. Otherwise, a default configuration file provided in the UPP repository will be used.
+``USE_CRTM``: (Default: "FALSE")
+   Flag that defines whether external :term:`CRTM` coefficient files have been staged by the user in order to output synthetic satellite products available within the :term:`UPP`. If this is set to "TRUE", then the workflow will check for these files in the directory ``CRTM_DIR``. Otherwise, it is assumed that no satellite fields are being requested in the UPP configuration.
 
-``CUSTOM_POST_CONFIG_FP``: (Default: “”)
-   The full path to the custom post flat file, including filename, to be used for post-processing. This is only used if ``CUSTOM_POST_CONFIG_FILE`` is set to "TRUE".
+``CRTM_DIR``: (Default: "")
+   This is the path to the top CRTM fix file directory. This is only used if ``USE_CRTM`` is set to "TRUE".
+
+Ensemble Model Parameters
+============================
+
+``DO_ENSEMBLE``: (Default: "FALSE")
+   Flag that determines whether to run a set of ensemble forecasts (for each set of specified cycles).  If this is set to "TRUE", ``NUM_ENS_MEMBERS`` forecasts are run for each cycle, each with a different set of stochastic seed values. When "FALSE", a single forecast is run for each cycle.
+
+``NUM_ENS_MEMBERS``: (Default: "1")
+   The number of ensemble members to run if ``DO_ENSEMBLE`` is set to "TRUE". This variable also controls the naming of the ensemble member directories. For example, if ``NUM_ENS_MEMBERS`` is set to "8", the member directories will be named *mem1, mem2, ..., mem8*.  If it is set to "08" (with a leading zero), the member directories will be named *mem01, mem02, ..., mem08*. However, after reading in the number of characters in this string (in order to determine how many leading zeros, if any, should be placed in the names of the member directories), the workflow generation scripts strip away those leading zeros. Thus, in the variable definitions file (``GLOBAL_VAR_DEFNS_FN``), this variable appears with its leading zeros stripped. This variable is not used unless ``DO_ENSEMBLE`` is set to "TRUE".
+
+
+Stochastic Physics Parameters
+================================
+
+..
+   COMMENT: Move all physic params up with CCPP section?
+
+For the most updated and detailed documentation of these parameters, see the `UFS Stochastic Physics Documentation <https://stochastic-physics.readthedocs.io/en/latest/namelist_options.html>`__.
+
+
+``NEW_LSCALE``: (Default: "TRUE") 
+   #
+
+..
+   COMMENT: Cannot find information on NEW_LSCALE variable in the stochastic physics docs. 
```

Comment:
I will add documentation for NEW_LSCALE in the stochastic physics documentation.  
NEW_LSCALE:  is True, use correct formula for converting a spatial legnthscale into spectral space. The original formula has a bug in it, but is being kept for backwards compatibility.